hack/verify-examples-kind.sh

```diff
@@ -53,7 +53,7 @@ kind create cluster --name "${CLUSTER_NAME}" --kubeconfig "${KUBECONFIG}" || res
 
 ##### Test v1alpha2 CRD apply and that invalid examples are invalid.
 # Install CRDs
-kubectl apply --kubeconfig "${KUBECONFIG}" -f config/crd/v1alpha2 || res=$?
+kubectl apply --kubeconfig "${KUBECONFIG}" -f config/crd/stable || res=$?
```

Comment:
Feels like we probably should still be testing that the experimental CRDs apply successfully? That does kind of imply that we should have a test corpus of YAMLs that tests the experimental CRD fields in addition to the stable ones as well though.

May be outside the scope of this PR, I guess.

Reply:
Updated, thanks!